assets/zstandard/main.py

```
@@ -0,0 +1,111 @@
+        except subprocess.CalledProcessError:
+            failed = True
+        
+        if failed:
+            logger.warning("Decompression failed")
+
+        ingestout = self.output[self.dataset_name][json.dumps(self.properties)]["ingest"]
+
+        ingestout["decompress_time_taken_s"] = self.bench_info['time_taken']
+        ingestout["decompress_memory_average_B"] = self.bench_info['memory_average']
+        ingestout["decompress_failed"] = "yes" if failed else "no"
+        ingestout["decompress_start_time"] = datetime.datetime.fromtimestamp(self.bench_info['start_time']).strftime('%Y-%m-%d %H:%M:%S')
```

Ruff (0.12.2)

`datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)